### PR TITLE
Editorial: Enhance Windows Installation Instructions

### PR DESCRIPTION
**Pull request by snuggs**
Addresses:
    - https://github.com/w3c/web-platform-tests/issues/6876#issuecomment-334187748

  - [x] Fix misspellings in WIndows installation
  - [x] Better Windows examples as per @domenic's request

---
This change is [Reviewable](https://reviewable.io/reviews/w3c/web-platform-tests/8634)

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2017-12-08 14:31:31
Finished: 2017-12-08 14:37:32


<h2>Failing Jobs</h2>
<ul>

<li>chrome:unstable</li>

<li>MicrosoftEdge:14.14393</li>

</ul>




View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/21671)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/313512125)

File: README.md [hunk 1]
```diff
@@ -53,8 +53,15 @@ Running Tests Manually
...
 The test server can be started using
+```
+./wpt serve
+```
 
-    ./wpt serve
+**On Windows**: You will need to preceed the prior command with
+`python` or the path to the python binary.
+```bash
+c:\python27\python.exe wpt serve
```
Comment (domenic) [inline README.md]: Just do `python` here and elsewhere.

**Comment by snuggs** [comment]
@domenic Fixed in https://github.com/w3c/web-platform-tests/pull/8634/commits/44f23002a745631a455bc6e6a441cdfc8ee9697c

**Review by domenic — APPROVED** [review]
LGTM. Will give it a day or so before merging in case @jgraham has any thoughts; I believe he wrote this originally.

**Review by gsnedders — APPROVED** [review]

**Pull request merged**